diff --git a/node_modules/maplibre-gl-js-amplify/lib/cjs/AmplifyGeofenceControl/AmplifyMapDraw.js b/node_modules/maplibre-gl-js-amplify/lib/cjs/AmplifyGeofenceControl/AmplifyMapDraw.js
index 214cb0a..2b95178 100644
--- a/node_modules/maplibre-gl-js-amplify/lib/cjs/AmplifyGeofenceControl/AmplifyMapDraw.js
+++ b/node_modules/maplibre-gl-js-amplify/lib/cjs/AmplifyGeofenceControl/AmplifyMapDraw.js
@@ -4,10 +4,10 @@ var __importDefault = (this && this.__importDefault) || function (mod) {
 };
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.AmplifyMapDraw = void 0;
-const mapbox_gl_draw_1 = __importDefault(require("@mapbox/mapbox-gl-draw"));
-const maplibre_gl_draw_circle_1 = require("maplibre-gl-draw-circle");
-const geofenceUtils_1 = require("../geofenceUtils");
-const constants_1 = require("../constants");
+const mapbox_gl_draw_1 = __importDefault(await import("@mapbox/mapbox-gl-draw"));
+const maplibre_gl_draw_circle_1 = await import("maplibre-gl-draw-circle");
+const geofenceUtils_1 = await import("../geofenceUtils");
+const constants_1 = await import("../constants");
 class AmplifyMapDraw {
     constructor(map, ui) {
         this._mapBoxDraw = new mapbox_gl_draw_1.default({
